### ISSUE TITLE
Screen reader compatibility

### ISSUE THREAD

**Issue by yaxu** [tidalcycles/strudel#691]
It'd be great to test strudel with screen readers.

A workshop participant found that Strudel doesn't work with [VoiceOver](https://en.wikipedia.org/wiki/VoiceOver) on a mac. Some things seemed to work but basic moving forward or backwards one word at a time didn't. It seemed like strudel was stopping the keyboard shortcut from working? I think it was control-option-right to read next item and control-option-left to read previous item that they were trying to use to read the words in the editor.
